tests/thread/oethread_enc/enc.cpp

```diff
@@ -273,5 +312,3 @@ OE_SET_ENCLAVE_SGX(
     512,  /* HeapPageCount */
     512,  /* StackPageCount */
     16);  /* TCSCount */
```

Comment:
@EMumau - I had to resize the enclaves to make this test PASS on Windows. My changes are in master but the directory structure is different after Radhika modified it. Here is the version in the master i.e. my change:
https://github.com/Microsoft/openenclave/blob/master/tests/thread/enc/enc_oethread.cpp
OE_SET_ENCLAVE_SGX(
    1,    /* ProductID */
    1,    /* SecurityVersion */
    true, /* AllowDebug */
    128,  /* HeapPageCount */
    16,   /* StackPageCount */
    16);  /* TCSCount */

Reply:
ok, I see from the conflicts below that you still haven't merged to master. Great.